demo/android/Activity/picovoice-activity-demo-app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,87 @@
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
+
+    <TextView
+        android:id="@+id/wakeWordName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/wake_word_name"
+        android:layout_marginTop="20dp"
+        android:layout_marginStart="10dp"
+        android:textSize="14sp"
+        android:textColor="@android:color/black"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toEndOf="@id/phraseLabel"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
+
+    <TextView
+        android:id="@+id/contextLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/context_label"
+        android:layout_marginTop="10dp"
+        android:layout_marginStart="32dp"
+        android:textSize="14sp"
+        app:layout_constraintTop_toBottomOf="@id/wakeWordName"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
+
+    <TextView
+        android:id="@+id/contextName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/context_name"
+        android:layout_marginTop="10dp"
+        android:layout_marginStart="10dp"
+        android:textSize="14sp"
+        android:textColor="@android:color/black"
+        app:layout_constraintTop_toBottomOf="@id/wakeWordName"
+        app:layout_constraintStart_toEndOf="@+id/phraseLabel"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
+
+    <TextView
+        android:id="@+id/phraseLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/phrase_label"
+        android:layout_marginTop="10dp"
+        android:layout_marginStart="32dp"
+        android:textSize="14sp"
+        app:layout_constraintTop_toBottomOf="@+id/contextName"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
+
+    <TextView
+        android:id="@+id/phraseText"
+        android:layout_width="0sp"
+        android:layout_height="wrap_content"
+        android:text="@string/phrase_text"
+        android:layout_marginTop="10dp"
+        android:layout_marginStart="10dp"
+        android:textSize="14sp"
+        android:textColor="@android:color/black"
+        app:layout_constraintTop_toBottomOf="@+id/contextName"
+        app:layout_constraintStart_toEndOf="@+id/phraseLabel"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0" />
```

Comment:
[text info, Picovoice output box, button] would be a better layout at this point. Same for Rhino.